test/drivers.js

```diff
@@ -14,7 +14,77 @@ drivers.push({
+              'AS ' + 
+              'BEGIN ' +
+                'DECLARE \@i INT ' +
+
+                'IF OBJECT_ID(\'#mss_test\', \'U\') IS NOT NULL DROP TABLE #mss_test ' +
+                'IF OBJECT_ID(\'tempdb..#mss_test\') IS NOT NULL DROP TABLE #mss_test ' +
+                'CREATE TABLE #mss_test (id int not null identity(1, 1) primary key, name NVARCHAR(50)) ' +
+
+                'SET \@i = 1 ' +
+
+                'WHILE (\@i < 10) ' +
+                'BEGIN ' +
+                  'INSERT INTO #mss_test (name) VALUES (\'test\' + CONVERT(char(1), \@i)) ' +
+                  'SET \@i = \@i + 1; ' +
+                'END ' +
+                
+                'SET NOCOUNT ON; ' +
+                'SELECT * FROM #mss_test; ' +
+              'END',
+  dropSP2:'IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N\'[dbo].[SP_TEST2]\') AND type in (N\'P\', N\'PC\')) ' +
+          'DROP PROCEDURE [dbo].[SP_TEST2]',              
+  createSP2: 'CREATE PROCEDURE [dbo].[SP_TEST2]' +
+              '\@id int ' +
+              'AS ' + 
+              'BEGIN ' +
+                'DECLARE \@i INT ' +
+
+                'IF OBJECT_ID(\'#mss_test\', \'U\') IS NOT NULL DROP TABLE #mss_test ' +
+                'IF OBJECT_ID(\'tempdb..#mss_test\') IS NOT NULL DROP TABLE #mss_test ' +
+                'CREATE TABLE #mss_test (id int not null identity(1, 1) primary key, name NVARCHAR(50)) ' +
+
+                'SET \@i = 1 ' +
+
+                'WHILE (\@i < 10) ' +
+                'BEGIN ' +
+                  'INSERT INTO #mss_test (name) VALUES (\'test\' + CONVERT(char(1), \@i)) ' +
+                  'SET \@i = \@i + 1; ' +
+                'END ' +
+                
+                'SET NOCOUNT ON; ' +
+                'SELECT * FROM #mss_test WHERE id = \@id ;' +
+              'END',
+  dropSP3:'IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N\'[dbo].[SP_TEST3]\') AND type in (N\'P\', N\'PC\')) ' +
+          'DROP PROCEDURE [dbo].[SP_TEST3]',                
+  createSP3: 'CREATE PROCEDURE [dbo].[SP_TEST3]' +
+              '\@id int, ' +
+              '\@result varchar(20) OUTPUT ' +
+              'AS ' + 
+              'BEGIN ' +
+                'DECLARE \@i INT ' +
+
+                'IF OBJECT_ID(\'#mss_test\', \'U\') IS NOT NULL DROP TABLE #mss_test ' +
+                'IF OBJECT_ID(\'tempdb..#mss_test\') IS NOT NULL DROP TABLE #mss_test ' +
+                'CREATE TABLE #mss_test (id int not null identity(1, 1) primary key, name NVARCHAR(50)) ' +
+
+                'SET \@i = 1 ' +
+
+                'WHILE (\@i < 10) ' +
+                'BEGIN ' +
+                  'INSERT INTO #mss_test (name) VALUES (\'test\' + CONVERT(char(1), \@i)) ' +
+                  'SET \@i = \@i + 1; ' +
+                'END ' +
+                
+                'SET NOCOUNT ON; ' +
+                'SELECT \@result = name FROM #mss_test WHERE id = \@id ' +
+              'END'
+});        
+
```

Comment:
Created separate files for each sql script, you can find those files in test/sql/ directory.